optuna/multi_objective/_hypervolume/base.py

```diff
@@ -4,11 +4,50 @@
 
 
 class BaseHypervolume(object, metaclass=abc.ABCMeta):
-    """Base class for hypervolume calculators"""
+    """Base class for hypervolume calculators.
+
+    .. note::
+        This class is used for computing the hypervolumes of points in multi-objective space.
+        Each coordinate of each point represents one value of the multi-objective function.
+
+    .. note::
+        We check that each objective is to be minimized. Transform objective values that are
+        to be maximized before calling this class's `compute` method.
```

Comment:
A nit, but I think we need one more pair of backticks.
```suggestion
        to be maximized before calling this class's ``compute`` method.
```
Current
![image](https://user-images.githubusercontent.com/3255979/91439975-d857bd80-e8a8-11ea-9029-cdc1603a05fc.png)

Suggestion
![image](https://user-images.githubusercontent.com/3255979/91440270-51571500-e8a9-11ea-928d-5efa7c361455.png)